src/projects.js

```diff
@@ -0,0 +1,202 @@
+/*
+Get Subjects For Projects
+This script pulls every Subject from a list of Projects, then creates a new
+Datasette-ready database for that project.
+
+Inputs/configurables:
+- (hardcoded) PROJECTS: list of target projects
+- (hardcoded) OUTPUT_DIR: Output directory
+- (hardcoded) TABLE_PREFIX: Prefix for database table name
+
+Output:
+- A Datasette-ready database for each project, named project-{project_id}
+  (e.g. project-1234)
+
+Notes:
+- Currently used by 2023 Community & Crowds's Community Catalog
+  (https://github.com/zooniverse/community-catalog)
+*/
+
+const fs = require('fs')
+const { unparse } = require('papaparse')
+
```

Comment:
```suggestion

const fetchWithRetry = require('./fetchWithRetry')
```

Panoptes drops connections pretty frequently so retrying is useful.